app/assets/stylesheets/components.scss

```diff
@@ -292,6 +292,10 @@
   font-size: 22px;
   position: relative;
 
+  div {
+    right: -30px;
```

Comment:
This looks more like a hack, have you identified why the behaviour is different in chore and firefox and tested solution both browsers ?